packages/patternfly-4/react-core/src/components/DataList/DataListToggle.js

```diff
@@ -8,13 +8,15 @@ import { Button } from '../Button';
 const DataListToggle = ({
   className,
   isExpanded,
+  'aria-controls': ariaControls,
   'aria-label': ariaLabel,
   'aria-labelledby': ariaLabelledBy,
+  rowid,
   id,
   ...props
 }) => (
     <div className={css(styles.dataListToggle, className)} {...props}>
-      <Button id={id} variant="plain" aria-label={ariaLabel} aria-labelledby={ariaLabelledBy} aria-expanded={isExpanded}>
+      <Button id={id} variant="plain" aria-controls={ariaControls !== '' && ariaControls} aria-label={ariaLabel} aria-labelledby={ariaLabel !== "Details" ? null : `${rowid} ${id}`} aria-expanded={isExpanded}>
```

Comment:
can you import and use the ButtonVariant object from Button and use that instead of using "plain".

Reply:
I added a new commit with this change.